docs/src/main/asciidoc/_high-availability/Clustering_and_Domain_Setup_Walkthrough.adoc

```diff
@@ -161,12 +158,37 @@ config to:
     <interface name="public">
        <inet-address value="${jboss.bind.address:10.211.55.7}"/>
     </interface>
-    <interface name="unsecured">
-       <inet-address value="10.211.55.7" />
-    </interface>
 </interfaces>
 ----
 
+We also need to add the native management interface in the management-interfaces section.
+The default section for management-interfaces is
+
+[source,xml,options="nowrap"]
+----
+<management-interfaces>
+    <http-interface http-authentication-factory="management-http-authentication">
+        <http-upgrade enabled="true" sasl-authentication-factory="management-sasl-authentication"/>
+        <socket interface="management" port="${jboss.management.http.port:9990}"/>
+    </http-interface>
+</management-interfaces>
+----
+
+We need to add the native-interface.
+
+[source,xml,options="nowrap"]
+----
+<management-interfaces>
+    <http-interface http-authentication-factory="management-http-authentication">
+        <http-upgrade enabled="true" sasl-authentication-factory="management-sasl-authentication"/>
+        <socket interface="management" port="${jboss.management.http.port:9990}"/>
+    </http-interface>
+    <native-interface>
+        <socket interface="management" port="${jboss.management.native.port:9999}"/>
+    </native-interface>
+</management-interfaces>
+----
+
```

Comment:
@bangert @rhusar I see why this section is needed. It shouldn't be necessary to add the native interface to run a domain.